packages/@aws-cdk-containers/ecs-service-extensions/test/assign-public-ip.integ.snapshot/tree.json

```diff
@@ -642,120 +642,13 @@
                   }
                 },
                 "constructInfo": {
-                  "fqn": "@aws-cdk/core.CfnResource",
-                  "version": "0.0.0"
-                }
-              },
-              "ExecutionRole": {
```

Comment:
why was all this deleted?

Reply:
guess the tree has changed since this was last updated. probably not worth really diving into more than that.